### PR TITLE
Add Deutsche Telekom to ADOPTERS.md

### PR DESCRIPTION
**Pull request by daniel-eder**
As discussed today with @tsteenbe, we'd like to show that Deutsche Telekom is using ORT.
Please let me know if you need any changes to the description!

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ This really helps the project to gain momentum and credibility. It's a small con
 | Organization | Contact | Description of Use |
 |--------------|---------|--------------------|
 | Sovereign Cloud Stack | https://scs.community/ | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset](https://scs.community/about/#technological-vision). |
+| Deutsche Telekom AG | https://www.telekom.de/ | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our ORT fork at https://github.com/telekom/ort. |
```
Comment (tsteenbe) [inline ADOPTERS.md:9]: @daniel-eder  For contact I would prefer an email address or a GitHub username (daniel-eder) - we should fix this for SCS as well

**Review by tsteenbe — CHANGES_REQUESTED** [review]

File: ADOPTERS.md (hunk 1 above)
Comment (sschuberth) [reply ADOPTERS.md:9]: Ping @itrich. Maybe also have an extra URL column then?

Comment (daniel-eder) [reply ADOPTERS.md]: For now I replaced the URL with an e-mail address (as I don't think there's a good way to establish contact via github username).

Comment (daniel-eder) [reply ADOPTERS.md]: Done for the DT entry

Comment (tsteenbe) [reply ADOPTERS.md]: @daniel-eder  Are you Ok with me squashing your 3 commits into a single one?

Comment (daniel-eder) [reply ADOPTERS.md]: @tsteenbe of course!

**Review by tsteenbe — APPROVED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**